extensions/amp-analytics/0.1/test/test-requests.js

```diff
@@ -620,7 +678,7 @@ describes.realWin('Requests', {amp: 1}, (env) => {
         element
       );
       return replacePromise.then((replace) => {
-        expect(replace).to.equal('test e1=TESTE1&e2=teste2 foo');
+        expect(replace).to.equal('test e2=teste2&e1=TESTE1 foo');
```

Comment:
reordering should be fine, but may break more tests where string comparison is used. 
Do you think it's worth keeping the order? It should be fairly straightforward to do.

Reply:
I think it got reordered because of this change; i.e. creating new obj and so serializing it changed. Does that seem plausible?

Reply:
You're right. The reason is because now `urlReplacements.expandStringAsync(value, bindings, opt_allowlist)` resolves asynchronously. To enforce the order all we need to do is to declare the variable and assign its value to undefined beforehand. The question is it is worth keeping the order?